frontends/web/src/api/account.ts

```diff
@@ -342,7 +342,14 @@ export const testRegister = (pin: string): Promise<null> => {
   return apiPost('test/register', { pin });
 };
 
-export const connectKeystore = (code: AccountCode): Promise<{ success: boolean; }> => {
+export type TResultConnectKeystore = {
+  success: boolean;
+  errorCode?: 'wrongKeystore';
+};
```

Comment:
Good point. I improved the typing, but I could not make the errorCode not optional, as it may happen that the backend only returns success=false without any error code. E.g. when a user cancels the current keystore request

Reply:
We don't need it now, but maybe the backend should return 'canceled' or similar?

Reply:
I added the 'userAbort' errorCode in the next PR I drafted (see my other comment https://github.com/digitalbitbox/bitbox-wallet-app/pull/2487#issuecomment-1893403206). Anyway, the backend function can still returns unexpected errors without a specific error code. The only way to always have a defined error code is to wrap all the unexpected errors around a 'genericError' error code, but I don't know if that would be actually useful.